src/SFA.DAS.AssessorService.Web.Staff/Views/Search/Select.cshtml

```diff
@@ -21,103 +21,115 @@
+                    <td class="bold">Funding model</td>
+                    <td>@Model.Learner.FundingModel</td>
+                </tr>
+            </tbody>
+        </table>
     </div>
 </div>
 
 @if (Model.Learner.CertificateReference != null)
 {
     <div class="grid-row">
         <div class="column-two-thirds">
-            <h2 class="heading-medium">Certificate details</h2>
-            <div class="form-group">
-                <ul>
-                    <li>
-                        <p><span class="bold">Certificate number:</span> @Model.Learner.CertificateReference</p>
-                    </li>
-                    <li>
-                        <p><span class="bold">EPAO:</span> @Model.Learner.OrganisationName</p>
-                    </li>
-                    <li>
-                        <p><span class="bold">Standard:</span> @Model.Learner.Standard (@Model.Learner.StandardCode)</p>
-                    </li>
-                    <li>
-                        <p><span class="bold">Level:</span> @Model.Learner.Level</p>
-                    </li>
-                    <li>
-                        <p><span class="bold">Option:</span> @Model.Learner.Option</p>
-                    </li>
-                    <li>
-                        <p><span class="bold">Grade:</span> @Model.Learner.OverallGrade</p>
-                    </li>
-                    <li>
-                        <p><span class="bold">Learner start date:</span> @Model.Learner.LearnStartDate?.ToShortDateString()</p>
-                    </li>
-                    <li>
-                        <p><span class="bold">Achievement date:</span> @Model.Learner.AchievementDate?.ToShortDateString()</p>
-                    </li>
-                    <li>
-                        <p><span class="bold">Status:</span> @Model.Learner.CertificateStatus</p>
-                    </li>
-                </ul>
-            </div>
+            <table>
+                <caption class="heading-large">Certificate details</caption>
+                <tbody>
+                    <tr>
+                        <td><span class="bold">Certificate number</span></td>
+                        <td>@Model.Learner.CertificateReference</td>
+                    </tr>
+                    <tr>
+                        <td><span class="bold">EPAO</span></td>
+                        <td>@Model.Learner.OrganisationName</td>
+                    </tr>
+                    <tr>
+                        <td><span class="bold">Standard</span></td>
+                        <td>@Model.Learner.Standard (@Model.Learner.StandardCode)</td>
+                    </tr>
+                    <tr>
+                        <td><span class="bold">Option</span></td>
+                        <td>@Model.Learner.Option</td>
+                    </tr>
+                    <tr>
+                        <td><span class="bold">Level</span></td>
+                        <td>@Model.Learner.Level</td>
+                    </tr>
```

Comment:
@gdbarnes It looks like you've changed the order of Standard, Level, Option to Standard, Option, Level.  Is this intentional? :)

Reply:
I didn't think I had. Probably something to do with fixing the merge conflicts.